src/Controller/Api/DataCenterController.php

```diff
@@ -157,7 +157,7 @@ public function getCollectionAction(Request $request)
     /**
      * Get a data center for a given id.
      *
-     * @param string $id The id of the Data center.
+     * @param integer $id The id of the Data center.
```

Comment:
I changed this from string to integer, with the assumption that an ID always an integer is.

Reply:
This will be a refactor.